### PR TITLE
console.log -> debug

### PR DESCRIPTION
**Pull request by f3l1x**
Ref: https://github.com/zeit/now-builders/pull/854

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -39,7 +40,7 @@ export async function build({
     }
   } else {
     if (!(await ensureLocalPhp())) {
-      console.log(`
+      debug(`
```
Comment (ties-v) [inline src/index.ts]: Should a message like this appear only when `NOW_BUILDER_DEBUG` is defined? I mean, it is pretty essential that the user sees this. It is not like a debug message that you might want to see ;).

File: src/launchers/cgi.ts [hunk 2]
```diff
@@ -128,7 +129,7 @@ function query({ filename, path, host, headers, method, body }: PhpInput): Promi
     // PHP script execution end
     php.on('close', function (code, signal) {
       if (code !== 0) {
-        console.log(`🐘 PHP process closed code ${code} and signal ${signal}`);
+        debug(`🐘 PHP process closed code ${code} and signal ${signal}`);
```
Comment (ties-v) [inline src/launchers/cgi.ts]: Same as above: is this necessary or a debug message (a.k.a. is it optional for the user?)

File: src/launchers/server.ts [hunk 3]
```diff
@@ -30,7 +31,7 @@ async function startServer(filename: string): Promise<ChildProcess> {
   );
 
   server.on('close', function (code, signal) {
-    console.log(`🐘 PHP process closed code ${code} and signal ${signal}`);
+    debug(`🐘 PHP process closed code ${code} and signal ${signal}`);
```
Comment (ties-v) [inline src/launchers/server.ts]: Same as above, maybe only when `signal===0` this should be a debug message?

**Comment by f3l1x** [comment]
I decided to don't merge it. I think console.log is OK at this moment.

**Pull request closed without merging**